packages/radix-ui-themes/src/components/avatar.props.ts

```diff
@@ -1,23 +1,25 @@
-import { asChildProp, colorProp, highContrastProp, radiusProp } from '../props/index.js';
+import {
+  asChildPropDef,
+  colorPropDef,
+  highContrastPropDef,
+  radiusPropDef,
+} from '../props/index.js';
 import type { PropDef } from '../props/index.js';
 
 const sizes = ['1', '2', '3', '4', '5', '6', '7', '8', '9'] as const;
 const variants = ['solid', 'soft'] as const;
 
 const avatarPropDefs = {
-  ...asChildProp,
```

Comment:
I went 180° on my proposal to rename all `componentPropDefs` to `componentProps` and instead renamed all prop defs that didn't have the suffix because that added some conflicts in JSX when we renamed props when spreading (e.g. `asChild: asChildProp`)

Reply:
haha, there's always something we don't account for 😀